source/api_documentation/transactions_api/_signal_param_table.rst

```diff
@@ -16,11 +16,11 @@ Most of the fields in this table are now deprecated. See Custom Data & Signal Pa
 
   * - signal_name *(deprecated)*
```

Comment:
Isn't this being renamed to signal_names?

Reply:
`signal_names` is available for webhooks, but not for anything else yet. I'm going to leave this as is until all other use cases have also been renamed.